src/test/scenarios/datafactory/output/src/datafactory/azext_datafactory/generated/_params.py

```diff
@@ -59,17 +59,17 @@ def load_arguments(self, _):
 
     with self.argument_context('datafactory update') as c:
         c.argument('resource_group_name', resource_group_name_type)
-        c.argument('factory_name', options_list=['--name', '-n'], help='The factory name.')
+        c.argument('factory_name', options_list=['--name', '-n'], help='The factory name.', id_part='name')
         c.argument('tags', tags_type)
         c.argument('identity', action=AddIdentity, nargs='+', help='Managed service identity of the factory. Expect val'
                    'ue: KEY1=VALUE1 KEY2=VALUE2 ...')
 
     with self.argument_context('datafactory delete') as c:
         c.argument('resource_group_name', resource_group_name_type)
-        c.argument('factory_name', options_list=['--name', '-n'], help='The factory name.')
+        c.argument('factory_name', options_list=['--name', '-n'], help='The factory name.', id_part='name')
 
     with self.argument_context('datafactory configure-factory-repo') as c:
-        c.argument('location_id', help='The location identifier.')
+        c.argument('location_id', help='The location identifier.', id_part='name')

```

Comment:
This looks weird. What's the full path for the request url?

Reply:
https://github.com/Azure/autorest.az/blob/support-ids/src/test/scenarios/datafactory/input/datafactory.json#L75